### PR TITLE
Add headers parameters

### PR DESCRIPTION
**Pull request by aguillem**
Add a little code snippet to allow sending headers params on ajax call

### CODE REVIEW

**Comment by Serhioromano** [comment]
This one is nice and clean. Thank you.

**Comment by criztovyl** [comment]
This break/broke the demo, complains about the `this` part of `this.options.headers` is undefined, should be `self`.

File: js/calendar.js [hunk 1]
```diff
@@ -966,7 +968,8 @@ if(!String.prototype.formatNum) {
 							url: buildEventsUrl(source, params),
 							dataType: 'json',
 							type: 'GET',
-							async: false
+							async: false,
+							headers: this.options.headers,
```
Comment (criztovyl) [inline js/calendar.js]: `this` should be `self`?

**Comment by aguillem** [comment]
Sorry, i fix it and submit new pull request #667

**Pull request merged**